libctru/source/gfx.c

```diff
@@ -5,12 +5,13 @@
 #include <3ds/gfx.h>
 #include <3ds/svc.h>
 #include <3ds/linear.h>
+#include <3ds/mappable.h>
 #include <3ds/vram.h>
 
 GSP_FramebufferInfo topFramebufferInfo, bottomFramebufferInfo;
 
 u8 gfxThreadID;
-u8* gfxSharedMemory;
+u8* gfxSharedMemory = NULL;
```

Comment:
Setting it to NULL is not necessary since the variable lies in the `.bss` region which is set to zero.